data_acquisition/data_acquisition.py

```diff
@@ -0,0 +1,37 @@
+import requests
+import os
+import psycopg2
+import getpass
+
+# for connecting to remote database
+# conn = psycopg2.connect(database='yourdb', user='dbuser', password='abcd1234', host='server', port='5432', sslmode='require')
+# for creating local database initially
+# cur.execute("CREATE TABLE Jobs(Id SERIAL PRIMARY KEY, Title TEXT, Description TEXT, Company TEXT, Location TEXT, SalaryMin INTEGER, PostURL TEXT)")
+# con.commit()
+
+# make ZipRecruiter API call
+API_KEY = os.getenv("ZIPRECRUITER_API_KEY")
+try:
+  req = requests.get("https://api.ziprecruiter.com/jobs/v1?search=internship%20Job&days_ago=1&page=1&api_key=" + API_KEY)
+  response = req.json()
+  if response["success"] == True:
```

Comment:
It's more idiomatic, I believe, to do `if response["success"]:`.

Reply:
lol that was dumb of me